### PR TITLE
ACK service controllers RBAC issue fixed

### PR DESCRIPTION
**Pull request by vaibhavp**
Issue #353 

Description of changes:
As per the description, I have updated the file with instruction for the controller-gen RBAC command to create the role permissions to allow reads from ConfigMap and Namespace resources. 

Also regenerated controller for following services. 
- s3
- sns
- apigatewayv2
- elasticache
- ecr

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: services/apigatewayv2/pkg/resource/api/manager.go [hunk 1]
```diff
@@ -30,6 +30,8 @@ import (
 
 // +kubebuilder:rbac:groups=apigatewayv2.services.k8s.aws,resources=apis,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=apigatewayv2.services.k8s.aws,resources=apis/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups="",resources=namespaces,verbs=get;list;watch
```
Comment (jaypipes) [inline services/apigatewayv2/pkg/resource/api/manager.go:33]: ah, good catch on the `groups=""` thing :) I'd forgotten about that!

File: test/e2e/s3/cross-account.sh [hunk 2]
```diff
@@ -48,7 +48,6 @@ cat <<EOF | kubectl apply -f -
 apiVersion: s3.services.k8s.aws/v1alpha1
 kind: Bucket
 metadata:
-  namespace: testing
```
Comment (jaypipes) [inline test/e2e/s3/cross-account.sh:51]: Hmm, this seems unrelated. Can you comment why this change was necessary?

**Review by jaypipes — COMMENTED** [review]
Fantastic @vaibhavp! One little question inline for you, otherwise this looks ready to merge.

File: test/e2e/s3/cross-account.sh (hunk 2 above)
Comment (vaibhavp) [reply test/e2e/s3/cross-account.sh:51]: When we run the test for s3 service, at that time this resource look for testing namespace which is not present in out testing or we are not creating any such namespace in our testing script anywhere due to which test case for s3  was falling. SO fix this is I have removed this line so that Bucket resource can be created in default namespace or select name.

Comment (jaypipes) [reply test/e2e/s3/cross-account.sh:51]: Ah, OK.

@A-Hilaly, unless I'm mistaken, the cross-account resource management e2e test *requires* a Kubernetes `Namespace` to be created (and subsequently annotated), right? Was this e2e test assuming someone had already created a 'testing' `Namespace`? If so, we need to modify the test script to create the 'testing' `Namespace` and ensure it is annotated appropriately.

Comment (A-Hilaly) [reply test/e2e/s3/cross-account.sh:51]: Exactly, I already had a 'testing' `Namespace` on my cluster and i missed script the steps to create it..

> If so, we need to modify the test script to create the 'testing' Namespace and ensure it is annotated appropriately.

Yes, we will ideally need an annotated `namespace` to test the override/default account/region features
